tests/installation/zdup.pm

```diff
@@ -40,6 +40,11 @@ sub run() {
     my $defaultrepo;
     if (get_var('SUSEMIRROR')) {
         $defaultrepo = "http://" . get_var("SUSEMIRROR");
+        #SUSEMIRROR not set, zdup from ftp source for online migration
```

Comment:
I don't get the comment. `SUSEMIRROR` is clearly set in this codepath, it passed through if at line 41. Aren't you missing some if-else?